modules/security/src/main/java/org/eclipse/xpanse/modules/security/zitadel/config/UserRoleDescriptionCustomizer.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.security.zitadel.config;
+
+import io.swagger.v3.oas.models.Operation;
+import java.util.Objects;
+import org.eclipse.xpanse.modules.security.zitadel.common.RequiredRoleDescription;
+import org.springdoc.core.customizers.OperationCustomizer;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.stereotype.Component;
+import org.springframework.web.method.HandlerMethod;
+
+/**
+ * Custom Operation of Swagger-UI to show required roles.
+ */
+@Profile("zitadel")
+@Component
+public class UserRoleDescriptionCustomizer implements OperationCustomizer {
```

Comment:
@baixinsui do we really need this new annotation?
Can we not simply add the description when `secured` annotation is available in class or method level?

Reply:
I removed it and appended the required role to description of Operation.
![image](https://github.com/eclipse-xpanse/xpanse/assets/121531362/1d3999cf-0228-4cf8-b1b3-94bac924492d)